tests/includes/models/test-class-accredible-learndash-model-auto-issuance.php

```diff
@@ -351,9 +358,30 @@ public function test_validate_when_creating() {
 	}
 
 	/**
-	 * Test if it passes with valid data when updating.
+	 * Test if it passes with valid data when creating with lesson_completed.
 	 */
-	public function test_validate_when_updating() {
+	public function test_validate_when_creating_with_lesson_completed() {
+		$data = array(
+			'kind'                => 'lesson_completed',
+			'post_id'             => 1,
+			'accredible_group_id' => 1,
+			'created_at'          => time(),
+		);
+
+		try {
+			Accredible_Learndash_Model_Auto_Issuance::validate( $data );
+			$caught_exception = null;
+		} catch ( \Exception $error ) {
+			$caught_exception = $error->getMessage();
+		}
+
+		$this->assertNull( $caught_exception );
+	}
+
+	/**
+	 * Test if it passes with valid data when updating when kind is course_completed.
+	 */
+	public function test_validate_when_updating_when_kind_is_course_completed() {
```

Comment:
I'd name it `test_validate_when_updating_with_course_completed`.

Reply:
I had those changes locally but didn't save them. It's now renamed.